src/snowflake/connector/connection.py

```diff
@@ -251,6 +257,7 @@ def __init__(self, **kwargs):
         self._done_async_sfqids = set()
         self.telemetry_enabled = False
         self._session_parameters: Dict[str, Union[str, int, bool]] = {}
+        self.enable_stage_s3_privatelink_for_us_east_1 = False
```

Comment:
This is gonna be unnecessary once you add the getter for `enable_stage_s3_privatelink_for_us_east_1`